script/pop.js

```diff
@@ -0,0 +1,95 @@
+    tags: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstrap'],
+    seeLivelink: '#',
+    seeSourceLink: '#',
+  },
+
+  {
+    title: 'Multi-Post Stories',
+    src: './image/Snapshoot-Portfolio.svg',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.",
+    tags: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstrap'],
+    seeLivelink: '#',
+    seeSourceLink: '#',
+  },
+  {
+    title: 'Tonic',
+    src: './image/Snapshoot-3Portfolio.svg',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.",
+    tags: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstrap'],
+    seeLivelink: '#',
+    seeSourceLink: '#',
+  },
+  {
+    title: 'Multi-Post Stories',
+    src: './image/Snapshoot-4Portfolio.svg',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.",
+    tags: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstrap'],
+    seeLivelink: '#',
+    seeSourceLink: '#',
+  },
+];
+
+const popButtons = document.getElementsByClassName('view');
+const work = document.querySelector('.cards-radious');
+const popup = document.createElement('section');
+popup.classList.add('popup-body');
+
+function popupDisplay(cardNbr) {
+  popup.style.display = ('block');
+  work.appendChild(popup);
+  popup.innerHTML = `
+        <div class="tonic-popup">
+          <h2>${projects[cardNum].title}</h2>
+          <i id="closeBtn" class="fas fa-times" alt='#'></i>
+        </div>
+        <ul class="list-popup">
+          <h2 class="canopy">CANOPY</h2>
+          <li class="date"> <i class="fa fa-circle"></i>&#160; Back End Dev</li>
+          <li class="date"> <i class="fa fa-circle"></i>&#160; 2015</li>
+        </ul>
+        <div class="card-img">
+          <img src="${projects[cardNum].src}" alt="Project-preview-1">
+        </div>
+        <div class="media-img">
+        <img src="${projects[cardNum].src}" alt="Project-preview-1">
+      </div>
```

Comment:
The attribute of the "popupDeisplay" function is "cardNbr", while you are using "cardNum" inside the function which makes a bug while loading the popup window.
 ```suggestion
 // you can replace the function attribute by cardNum
function popupDisplay(cardNum) {
```